lib/webpackImporter.js

```diff
@@ -46,6 +46,9 @@ function webpackImporter(resourcePath, resolve, addNormalizedDependency) {
             Promise.reject() :
             resolve(dir, importsToResolve[0])
                 .then(resolvedFile => {
+                    if (!([".sass", ".scss", ".css"].includes(path.extname(resolvedFile)))) {
+                        throw new Error("Alias should have 'sass', 'scss' or 'css' extension");
```

Comment:
This requirement is very arbitrary and I don't think that we should introduce that. Why do aliases have to have a `sass`, `scss` or `css` extension? Furthermore, it's not necessarily an alias that caused this condition, so the error message is also misleading.

Reply:
@jhnns i know :disappointed: , it is very interesting. Found solution:
1.  Add here url + all paths from `includePaths` https://github.com/webpack-contrib/sass-loader/blob/master/lib/importsToResolve.js#L56 (i.e. `includePath[0]/${url}`, `includePath[1]/${url}`, url)
2. Node-sass should be try import first when we try import throw own implement (not sure it is possible)

Any ideas how we can fix this?

Reply:
Currently not. I wish we could let node-sass handle the imports as default and then use webpack's resolver, but that's not possible with the current node-sass API.

Reply:
@jhnns :disappointed: I try to search solution todat, have some ideas.